stable/sealed-secrets/templates/deployment.yaml

```diff
@@ -16,6 +16,12 @@ spec:
       app.kubernetes.io/instance: {{ .Release.Name }}
   template:
     metadata:
+      {{- if .Values.podAnnotations }}
```

Comment:
I guess you could get rid of the `if` all together, since the `with` basically handles the condition already?

Reply:
Maybe, but I guess in that case we'll always end up with `.spec.template.annotations` (explicitly) equal to `{}` whenever `.Values.podAnnotations` is not defined. There's probably no problem in that so I'll amend the PR.